tests/unit/converters/test_line_of_sight.py

```diff
@@ -2,193 +2,136 @@
 def test_convert_to_Rz(debug=False):
     # Load line-of-sight default
-    los, machine_dims = load_los_default()
-    x1, x2, t = default_inputs()
+    los, machine_dims = load_line_of_sight_default()
+    x1 = 0
+    x2 = los.x2
+    t = 0
 
     # Test method
-    for ind, iLoS in enumerate(los):
-        R_, z_ = iLoS.convert_to_Rz(x1, x2, t)
-
-        x, y = iLoS.convert_to_xy(x1, x2, t)
-        R = np.sqrt(x**2 + y**2)
-
-        # R and z are as expected=
-        assert np.allclose(R, R_)
-
-        if debug:
-            print(f"R = {R}")
+    R_, z_ = los.convert_to_Rz(x1, x2, t)
 
+    x, y = los.convert_to_xy(x1, x2, t)
+    R = np.sqrt(x**2 + y**2)
 
-# Test convert_from_Rz method
-def test_convert_from_Rz(debug=False):
-    # Load line-of-sight default
-    los, machine_dims = load_los_default()
-
-    # Test input
-    x1, x2, t = default_inputs()
-
-    # Test method
-    for ind, iLoS in enumerate(los):
-        print(iLoS.name)
-        R_test, Z_test = iLoS.convert_to_Rz(x1, x2, t)
-        _, x2_out = iLoS.convert_from_Rz(R_test[2], Z_test[2], t)
-
-        # x2 is within specified range
-        assert (x2_out < 1 or np.isclose(x2_out, 1)) and (
-            x2_out > 0 or np.isclose(x2_out, 0)
-        )
+    # R and z are as expected=
+    assert all(R == R_)
 
-        if debug:
-            print(f"x2_out2 = {x2_out}")
+    if debug:
+        print(f"R = {R}")
 
 
 # Test distance method
 def test_distance(debug=False):
     # Load line-of-sight default
-    los, machine_dims = load_los_default()
-    x1, x2, t = default_inputs()
+    los, machine_dims = load_line_of_sight_default()
+    x1 = 0
+    x2 = los.x2
+    t = 0
 
     # Test method
-    for ind, iLoS in enumerate(los):
-        dist = iLoS.distance("dim_0", x1, x2, t)
-        dls = [dist[i + 1] - dist[i] for i in range(len(dist) - 1)]
+    dist = los.distance("los_position", x1, x2, t)
+    dls = [dist[i + 1] - dist[i] for i in range(len(dist) - 1)]
 
-        # dl is identical along the line of sight up to 1 per million
-        assert all(np.abs(dls - dls[0]) < (dls[0] * 1.0e-6))
+    # dl is identical along the line of sight up to 1 per million
+    assert all(np.abs(dls - dls[0]) < (dls[0] * 1.0e-6))
 
-        if debug:
-            print(f"dist = {dist}")
+    if debug:
+        print(f"dist = {dist}")
+    return
 
 
 # Test distance method
 def test_set_dl(debug=False):
     # Load line-of-sight default
-    los, machine_dims = load_los_default()
+    los, machine_dims = load_line_of_sight_default()
 
     # Test inputs
     dl = 0.002
 
     # Test method
-    for ind, iLoS in enumerate(los):
-        x2, dl_out = iLoS.set_dl(dl)
+    los.set_dl(dl)
 
-        assert np.isclose(dl, dl_out, rtol=1e-2)
+    assert np.abs(dl - los.dl) < 1.0e-4
 
-        if debug:
-            print(f"x2 = {x2}")
-            print(f"dl_out = {dl_out}")
+    if debug:
+        print(f"x2 = {los.x2}")
+        print(f"dl_out = {los.dl}")
     return
 
 
```

Comment:
The unit test `test_intersections` can probably be moved to `tests/unit/test_utilities.py` now.